app/views/content/shared/qualifications-training/_check_qualifcations.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="check-qualifcations">
+  <h3>Check your qualifications</h3>
+  <p>To train to teach <%= @front_matter["subject"] %> in England, you'll need:</p>
+  <ul>
+    <li>GCSEs at grade 4 (C) or above in English and maths (and science if you want to teach primary)</li>
```

Comment:
```suggestion
    <li>GCSEs at grade 4 (C) or above in English and maths</li>
```
We don't need to refer to primary here if this is a partial only being used on secondary pages @Sarah-DfE @joegibb ?

Reply:
Think the idea here was to keep in general so we could use it on other pages. So in theory, this partial could be used across the site but without chemistry/subject in it.

Reply:
But on line 3 it says `<p>To train to teach <%= @front_matter["subject"] %> in England, you'll need:</p>` so you could only ever use this partial on a subject page?

Reply:
I thought we could remove the front matter bit, so it would say, 'To train to teach in England, you'll need' etc

Reply:
Ignore that - if you didn't have a subject in front matter, it would just say 'To train to teach in England....' :-)